test/Groups/gsets.jl

```diff
@@ -335,7 +335,7 @@ end
     GL = general_linear_group(n, F)
     S = sylow_subgroup(GL, 2)[1]
     for G in [GL, S]
-#     for k in 0:n   # k = 0 is a problem in GAP 4.12.0
+#     for k in 0:n   # k = 0 is a problem in GAP 4.14.0
```

Comment:
Meanwhile I think it is better to fix this on the Oscar side. I will create a pull request for that.